### ISSUE TITLE
lyrics plus for Spotify

### ISSUE THREAD

**Issue by Chingman12** [dupitydumb/LyricsPlus-Spotify#3]
Extension is not working even if I follow the instruction

**Comment by dupitydumb** [comment]
which version did you use? the chrome extension or spicetify?

**Comment by GildasEdraE** [comment]
> Extension is not working even if I follow the instruction

i got error 
![Screenshot (91)](https://github.com/dupitydumb/LyricsPlus-Spotify/assets/89395504/bccbfdec-2738-4b15-b5b1-5ad722b15fe2)

**Comment by GildasEdraE** [comment]
![Screenshot (92)](https://github.com/dupitydumb/LyricsPlus-Spotify/assets/89395504/3361116b-6254-47a2-9c90-b3438e13adb4)

**Comment by dupitydumb** [comment]
for a temporary fix you could manually refresh page after click the apply button

![2024-04-28 14-43-10](https://github.com/dupitydumb/LyricsPlus-Spotify/assets/37872714/ec8ae8f9-d315-4b05-ac48-a5a0711ecae5)

**Comment by vianneynara** [comment]
Might be a bit out of context, but does this modifier extension only work for the web version of Spotify? Is there any way for the desktop marketplace version to be modified?

**Comment by vianneynara** [comment]
![image](https://github.com/dupitydumb/LyricsPlus-Spotify/assets/98963440/2a2c5c77-c75a-4dfb-866b-33b4833972ea)
Background color modifiers are not working in the desktop app (Windows/Spicetify). Only the font size modifier work.

I can't see the lyrics well...

**Comment by GildasEdraE** [comment]
https://github.com/dupitydumb/LyricsPlus-Spotify/issues/2#issuecomment-2080386670

> ![image](https://private-user-images.githubusercontent.com/98963440/327527472-2a2c5c77-c75a-4dfb-866b-33b4833972ea.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ2NzIzMjIsIm5iZiI6MTcxNDY3MjAyMiwicGF0aCI6Ii85ODk2MzQ0MC8zMjc1Mjc0NzItMmEyYzVjNzctYzc1YS00ZGZiLTg2NmItMzNiNDgzMzk3MmVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTAyVDE3NDcwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY0OTE4YjVjMGY4M2E4ZWQwYWUwOWQ3NzA0ZDVhZDgzNjFhMThkNDI1ZTRhMzI0MzhjOGFlNDgyMWUzNzI5ZmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.F-BcOdtiEIKgN9-7o3WoC_8s5AniVrkTVHW6L6IutEI) Background color modifiers are not working in the desktop app (Windows/Spicetify). Only the font size modifier work.
> 
> I can't see the lyrics well...

https://github.com/dupitydumb/LyricsPlus-Spotify/issues/2#issuecomment-2080386670

**Comment by dupitydumb** [comment]
the custom setting for spicetify is on the way

**Comment by dupitydumb** [comment]
@vianneynara @GildasEdraE @Chingman12 need some help to check something. did the extension disappear from spicetify marketplace?

**Comment by Rea-Lux-Null** [comment]
> @vianneynara @GildasEdraE @Chingman12 need some help to check something. did the extension disappear from spicetify marketplace?

![Screenshot (114)](https://github.com/dupitydumb/CustomLyrics-Spotify/assets/89395504/58ca6ecd-6593-4234-8290-1e18a36d2e6c)
You meaning this?

**Comment by dupitydumb** [comment]
> > @vianneynara @GildasEdraE @Chingman12 need some help to check something. did the extension disappear from spicetify marketplace?
> 
> ![Screenshot (114)](https://private-user-images.githubusercontent.com/89395504/329132700-58ca6ecd-6593-4234-8290-1e18a36d2e6c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUyMzgyNjAsIm5iZiI6MTcxNTIzNzk2MCwicGF0aCI6Ii84OTM5NTUwNC8zMjkxMzI3MDAtNThjYTZlY2QtNjU5My00MjM0LTgyOTAtMWUxOGEzNmQyZTZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA5VDA2NTkyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE3OWYyMjgxYmMxNzdiMGExY2Y5OGM5YTIzODYzNzk3MjY0MGEyYTBlZDAyYjNkYjMxYmRmN2EyMzkwYWIxNGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5eRCO0zFAh3Q_zyXJWEjwZ6cdj9ZrYvqZB6QCwArWGk) You meaning this?

yes. turnout last time it got removed.

**Comment by vianneynara** [comment]
Yes, it got removed from the market. My friend was unable to get it

**Comment by dupitydumb** [comment]
> Yes, it got removed from the market. My friend was unable to get it

yes it was remove on this spicetify pull request [1](https://github.com/spicetify/spicetify-marketplace/pull/742#issuecomment-2095934187) but it can be downloaded again after I change the name [2](https://github.com/spicetify/spicetify-marketplace/pull/745)

**Comment by Rea-Lux-Null** [comment]
If it is missing, you should add manual download method

**Comment by Rea-Lux-Null** [comment]
For Example
1. Install Spicetify
2. Download (create a file name).js
3. Navigate to your Spicetify config directory via the command `spicetify config-dir`
4. Place (create a file name).js in `/extension`
5. Run `spicetify config extension (create a name file).js` and `spicetify apply` in terminal or PowerShell(run as administrator)

**Comment by Rea-Lux-Null** [comment]
I think you add js file name create that as `content.js`
